<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="101" started-at="2016-07-08T11:36:14Z" finished-at="2016-07-08T11:36:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="101" started-at="2016-07-08T11:36:14Z" finished-at="2016-07-08T11:36:14Z">
      <class name="com.thalesgroup.services.cte.lis.dojo.Story7Test">
        <test-method status="FAIL" signature="testValueOneSubregionOnly()[pri:0, instance:com.thalesgroup.services.cte.lis.dojo.Story7Test@22a71081]" name="testValueOneSubregionOnly" duration-ms="80" started-at="2016-07-08T13:36:14Z" finished-at="2016-07-08T13:36:14Z">
          <exception class="java.security.InvalidParameterException">
            <message>
              <![CDATA[Tried to set value '4' for cell '(row=3,col=9)' but cell '(row=3,col=7)' already has it !]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.security.InvalidParameterException: Tried to set value '4' for cell '(row=3,col=9)' but cell '(row=3,col=7)' already has it !
	at com.thalesgroup.services.cte.lis.dojo.Cell.setValue(Cell.java:77)
	at com.thalesgroup.services.cte.lis.dojo.Cell.removeImpossibleValue(Cell.java:33)
	at com.thalesgroup.services.cte.lis.dojo.NineCells.NGroupsValuesLock(NineCells.java:165)
	at com.thalesgroup.services.cte.lis.dojo.Grid.NGroupsValuesLock(Grid.java:138)
	at com.thalesgroup.services.cte.lis.dojo.Grid.solve(Grid.java:187)
	at com.thalesgroup.services.cte.lis.dojo.Sudoku.solve(Sudoku.java:11)
	at com.thalesgroup.services.cte.lis.dojo.SudokuTest$SolvedProblem.<init>(SudokuTest.java:51)
	at com.thalesgroup.services.cte.lis.dojo.Story7Test.testValueOneSubregionOnly(Story7Test.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.security.InvalidParameterException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValueOneSubregionOnly -->
      </class> <!-- com.thalesgroup.services.cte.lis.dojo.Story7Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
